### PR TITLE
Sysrex bootstrappers

### PR DESCRIPTION
**Pull request by sysrex**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,14 +5,15 @@ COPY . /orchestrator-relayer
...
 RUN apk update && apk --no-cache add bash
 
 COPY --from=builder /orchestrator-relayer/build/qgb /bin/qgb
+COPY --chown=${USER_NAME}:${USER_NAME} docker/entrypoint.sh /opt/entrypoint.sh
 
 # p2p port
 EXPOSE 30000
 
-ENTRYPOINT [ "/bin/qgb" ]
+ENTRYPOINT [ "/bin/bash", "/opt/entrypoint.sh" ]
```
Comment (SweeXordious) [inline Dockerfile:19]:
```suggestion
ENTRYPOINT [ "/bin/bash", "/opt/entrypoint.sh" ]

```

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+
+echo "Starting Celestia QGB with command:"
+echo "$@"
+echo ""
+
+exec "$@"
```
Comment (SweeXordious) [inline docker/entrypoint.sh:10]:
```suggestion
exec "$@"

```

File: Dockerfile [hunk 3]
```diff
@@ -5,14 +5,15 @@ COPY . /orchestrator-relayer
 WORKDIR /orchestrator-relayer
 RUN make build
 
-# stage 2
-FROM alpine:3.17.3
+# final image
+FROM alpine:3.17.2
```
Comment (SweeXordious) [inline Dockerfile:9]: Why not keep `3.17.3`?

**Review by SweeXordious — COMMENTED** [review]
Thanks :rocket:

File: Dockerfile (hunk 3 above)
Comment (sysrex) [reply Dockerfile:9]: actually, this is the one present in the e2e Dockerfile that's why I kept it, I don't have strong considerations around minor versions in alpine.

Comment (SweeXordious) [reply Dockerfile:9]: let's leave it for now then, it's cool

**Review by SweeXordious — APPROVED** [review]

**Review by SweeXordious — APPROVED** [review]

**Pull request merged**